scripts/xtensa-build-all.sh

```diff
@@ -345,9 +379,11 @@ do
 		make overrideconfig
 	fi
 
-	if [ 'tgl' != "${platform}" ]; then
+	# TGL needs MEU arguments for signing
+	if [ 'tgl' == "${platform}" ] && [ -n "${MEU_PATH_OPTION}" ] \
```

Comment:
`==` is an unnecessary bashism inside a `[   ]` non-bashism.

Reply:
no idea what you mean here. A bash for dummies like comment will be helpful